composables/cache.ts

```diff
@@ -106,23 +83,13 @@ export async function fetchAccountByHandle(acct: string): Promise<mastodon.v1.Ac
       account.acct = `${account.acct}@${domain}`
     return account
   }
-  let fetchPromise = fetchPromises.get(key)
-  if (!fetchPromise) {
-    fetchPromise = lookupAccount()
-      .then((r) => {
-        cacheAccount(r, server, true)
-        return Promise.resolve(r)
-      })
-      .finally(() => fetchPromises.delete(key))
-    fetchPromises.set(key, fetchPromise)
-  }
-
-  const promise = new Promise<any>((resolve) => {
-    fetchPromise!.then(resolve)
-  })
-  if (!cache.has(key))
-    cache.set(key, promise)
 
+  const promise = lookupAccount()
+    .then((r) => {
+      cacheAccount(r, server, true)
```

Comment:
Same here. I think it is important that we make sure the `key` is the same, so we don't leave any promises in the cache.